src/renderer/containers/ExplorePage.tsx

```diff
@@ -120,7 +97,8 @@ function overridesCount(observations: Observation[]): number {
 
 export default function ExplorePage() {
   const { t } = useTranslation();
-  const [filePath, setFilePath] = useState<string>();
+  const [csvFilePath, setCsvFilePath] = useState<string>('');
+  const [dbDirPath, setDbDirPath] = useState<string>('');
```

Comment:
Perhaps `csvPath` and `photosPath` to keep things simple and descriptive? :slightly_smiling_face:

Reply:
done